docker-bake.hcl

```diff
@@ -6,28 +6,29 @@ variable "TAG" {
     default = "main"
 }
 
-target "cluster-api-provider-openstack-source" {
-    context = "images/source-patch"
-    platforms = ["linux/amd64", "linux/arm64"]
-
-    contexts = {
-        "git" = "https://github.com/kubernetes-sigs/cluster-api-provider-openstack.git#v0.11.6"
-        "patches" = "patches/kubernetes-sigs/cluster-api-provider-openstack"
-    }
-}
-
-target "cluster-api-provider-openstack" {
-    context = "images/cluster-api-provider-openstack"
-    platforms = ["linux/amd64", "linux/arm64"]
-
-    contexts = {
-        "source" = "target:cluster-api-provider-openstack-source"
-    }
-
-    tags = [
-        "${REGISTRY}/capi-openstack-controller:${TAG}"
-    ]
-}
+# NOTE(oleksandr1203): We keep this commented out for now as we'll need patch in the future.
+# target "cluster-api-provider-openstack-source" {
+#     context = "images/source-patch"
+#     platforms = ["linux/amd64", "linux/arm64"]
+
+#     contexts = {
+#         "git" = "https://github.com/kubernetes-sigs/cluster-api-provider-openstack.git#v0.12.4"
+#         "patches" = "patches/kubernetes-sigs/cluster-api-provider-openstack"
+#     }
+# }
+
+# target "cluster-api-provider-openstack" {
+#     context = "images/cluster-api-provider-openstack"
+#     platforms = ["linux/amd64", "linux/arm64"]
+
+#     contexts = {
+#         "source" = "target:cluster-api-provider-openstack-source"
+#     }
+
+#     tags = [
+#         "${REGISTRY}/capi-openstack-controller:${TAG}"
+#     ]
+# }
```

Comment:
Let's remove this instead of commenting it out, we can always check Git history to bring it back.